bundles/org.openhab.binding.opensprinkler/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -32,6 +32,18 @@
 				<description>Specifies the refresh interval in seconds.</description>
 				<default>60</default>
 			</parameter>
+			<parameter name="retry" type="integer">
```

Comment:
Please update the i18n properties file to add these 2 new parameters.

Reply:
To do it simply, run: mvn i18n:generate-default-tramslations

Reply:
To accelerate process and merge this PR before OH 4.1 is released, I run the command myself and push the change.